types/save-csv/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "npm-naming": false
```

Comment:
Why is this still disabled?

Reply:
Because I was waiting for your answer. Now it shouldn't be there anymore.